plotly/offline/offline.py

```diff
@@ -298,3 +309,128 @@ def plot(figure_or_data,
             ])
         else:
             return plot_html
+
+
+def plot_mpl(mpl_fig, resize=False, strip_style=False,
+             verbose=False, **kwargs):
+    '''
+    Convert a matplotlib figure to a plotly graph locally as an HTML document
+    or string
```

Comment:
:cow2: Can we make the one-liner a one-liner :smile_cat: 

`Convert a matplotlib figure to a Plotly graph stored locally as HTML.`